extra/innochecksum.cc

```diff
@@ -100,6 +100,8 @@ FILE*				log_file = NULL;
 /* Enabled for log write option. */
 static bool			is_log_enabled = false;
 
+/* Skip freed pages */
+static bool			skip_freed_pages= false;
```

Comment:
`static` and global variables are zero-initialized by the loader by default, by the virtue of assigning them to BSS. Modern compilers can optimize away the assignment, but older compilers might generate some code for the redundant zero-initialization.